Source/ThirdParty/EASTL/include/EASTL/string.h

```diff
@@ -934,6 +887,111 @@ namespace eastl
+
+		iterator Begin() { return begin(); }
+		iterator End() { return end(); }
+		const_iterator Begin() const { return begin(); }
+		const_iterator End() const { return end(); }
+		reference Front() { return front(); }
+		reference Back() { return back(); }
+		const_reference Front() const { return front(); }
+		const_reference Back() const { return back(); }
+		const_pointer CString() const { return c_str(); }
+
+		void Replace(value_type old_value, value_type new_value) { replace(old_value, new_value); }
+		void Replace(view_type old_value, view_type new_value) { replace(old_value, new_value); }
+    	void Replace(size_type position, size_type n, const this_type& x) { replace(position, n, x); }
+    	void Replace(size_type position, size_type n, const value_type* x) { replace(position, n, x); }
+		iterator Replace(const_iterator first, const_iterator last, const this_type& x)
+		{
+			size_type pos = first - begin();
+			replace(first, last, x);
+			return begin() + pos;
+		}
+		this_type Replaced(value_type old_value, value_type new_value) const { return replaced(old_value, new_value); }
+		this_type Replaced(view_type old_value, view_type new_value) const { return replaced(old_value, new_value); }
+
+		this_type& Append(const this_type& x) { return append(x); }
+		this_type& Append(const value_type* p) { return append(p); }
+		this_type& Append(const value_type* p, size_type n) { return append(p, n); }
+		this_type& Append(value_type x) { return append(&x, 1); }
+		this_type& AppendWithFormat(const char* pFormat, ...)
+		{
+			va_list args;
+			va_start(args, pFormat);
+			append_sprintf_va_list(pFormat, args);
+			va_end(args);
+			return *this;
+		}
+
+    	void Insert(size_type position, const this_type& x) { insert(position, x); }
+    	void Insert(size_type position, value_type x) { insert(position, 1, x); }
+    	iterator Insert(const_iterator p, const this_type& x) { return insert(p, x.begin(), x.end()); }
+		iterator Insert(const_iterator p, const value_type* pBegin, const value_type* pEnd) { return insert(p, pBegin, pEnd); }
+		iterator Insert(const_iterator p, value_type x) { return insert(p, x); }
+		void Erase(size_type position, size_type n = 1) { erase(position, n); }
+    	iterator Erase(const_iterator p) { return erase(p, p + 1); }
+    	iterator Erase(const_iterator pBegin, const_iterator pEnd) { return erase(pBegin, pEnd); }
+
+		void Reserve(size_type size) { reserve(size); }
+		void Resize(size_type size) { resize(size); }
+		void Resize(size_type size, value_type value) { resize(size, value); }
+		void Clear() { clear(); }
+		void Compact() { shrink_to_fit(); }
+		void Swap(this_type& other) { swap(other); }
+
+    	this_type Substring(size_type position, size_type n = npos) const { return substr(position, n); }
+		this_type Trimmed() const { return trimmed(); }
+		this_type ToUpper() const { return to_upper(); }
+		this_type ToLower() const { return to_lower(); }
```

Comment:
Tabs and spaces are mixed all over \o/

Reply:
Oh shit. I totally forgot about it. Sometimes I copy-pasted code from headers, sometimes I worked in-place.